collector/src/main/java/io/prometheus/jmx/JmxScraper.java

```diff
@@ -190,7 +190,12 @@ private void processBeanValue(
             Object value) {
         if (value == null) {
             logScrape(domain + beanProperties + attrName, "null");
-        } else if (value instanceof Number || value instanceof String || value instanceof Boolean) {
+        } else if (value instanceof Number || value instanceof String || value instanceof Boolean || value instanceof java.util.Date) {
+            //output Date as Long (the number of seconds since January 1, 1970, 00:00:00 GMT)
+            if(value instanceof java.util.Date){
```

Comment:
formatting is still off here.

Reply:
fixed formatting

Reply:
There should be a space after the if and before the {

Reply:
updated to desired style